arcgis-ios-sdk-samples/Content Display Logic/Controllers/ContentCollectionViewController.swift

```diff
@@ -45,7 +45,12 @@ class ContentCollectionViewController: UICollectionViewController, UICollectionV
         searchBar.autocapitalizationType = .none
         
         if #available(iOS 13, *) {
-            // Do nothing, on iOS 13 the settings below have no effect.
+            // Change the backgroundColor to cope with #1517 and #1326
```

Comment:
Please remove these issue numbers as the issues themselves are not publicly visible.